bcl/src/bc_lp8.c

```diff
@@ -5,7 +5,10 @@
 #define _BC_LP8_MODBUS_READ 0x44
 #define _BC_LP8_INITIAL_MEASUREMENT 0x10
 #define _BC_LP8_SEQUENTIAL_MEASUREMENT 0x20
-#define _BC_LP8_RX_ERROR_STATUS0 (3 + 39)
+#define _BC_LP8_RX_ERROR_STATUS0 (3 + 0xa7 - 0x80)
+#define _BC_LP8_RX_ERROR_STATUS1 (3 + 0xa6 - 0x80)
+#define _BC_LP8_RX_CONC (3 + 0x9a - 0x80)
```

Comment:
What is CONC?

Reply:
Datasheat LP8, page 17, table Parameters, row 4 : 
Conc : Non pressure-corrected unfiltered concentration value

Reply:
OK